ballot/app/package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "test": "echo \"Run tests here\""
```

Comment:
Punting in this PR because running jest here spews the error

```
 FAIL  test/service.spec.ts
  ● Test suite failed to run

    /Users/armaniferrante/Documents/code/src/github.com/oasislabs/tutorials/ballot/app/test/service.spec.ts:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import "regenerator-runtime/runtime";
                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    SyntaxError: Unexpected string

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.802s
```